src/frontend/sharedComponents/CustomHeaderLeftClose.tsx

```diff
@@ -243,19 +243,18 @@ const SharedBackButton = ({
 function checkEqual(
   original: Observation,
   {
-    editted,
-    numberOfPhotos,
+    edited,
+    numberOfAttachments,
   }: {
-    editted: Observation | ClientGeneratedObservation | null;
-    numberOfPhotos?: number;
+    edited: Observation | ClientGeneratedObservation | null;
+    numberOfAttachments?: number;
   },
 ) {
-  if (!editted || !('docId' in editted)) return true;
-  // attachments are created right before an observation is made, so we need to check # photos that are about to be saved
-  const {attachments: originalAtts, ...orignalNoPhotos} = original;
+  if (!edited || !('docId' in edited)) return true;
+  // attachments are created right before an observation is made, so we need to check # attachments that are about to be saved
+  const {attachments: originalAtts} = original;
 
-  if (originalAtts.length !== numberOfPhotos) return false;
+  if (originalAtts.length !== numberOfAttachments) return false;
```

Comment:
this is check is wrong now. number of attachments is no longer a good incdicator since audio files can be created and deleted

Reply:
I thought it was a good way of doing a quick check before we do the more involved check... If they aren't the same length then something definitely changed so we can return false right away before doing the isEqual check next.